src/Symfony/Component/HttpKernel/Kernel.php

```diff
@@ -536,6 +537,12 @@ protected function getKernelParameters()
             $bundles[$name] = get_class($bundle);
         }
 
+        try {
+            $logDir = realpath($this->getLogDir()) ?: $this->getLogDir();
+        } catch (DisableLogDirectoryException $e) {
+            $logDir = false;
```

Comment:
Here, the code is unable to decide between getLogDir returning false, or getLogDir throwing an exception.
I think we need some $hasLogDir variable that is false/true when log is throwing or not.